R/viewer.R

```diff
@@ -1,4 +1,80 @@
-htmlViewer = function(html) {
+COMPONENT_MAP = list(
+    'collection' = 'data',
+    'data-table' = 'data',
+    'file-format' = 'format',
+    'file-formats' = 'formats',
+    'item' = 'data',
```

Comment:
Maybe here is somethint amiss for `list_results()`

Reply:
No, should be complete. It works as it is in the Python client.